data/config/languages.lua

```diff
@@ -36,6 +36,10 @@ languages = {
         name = "Galego",
         interwords_spaces = true
     },
+    ["gd"] = {
```

Comment:
It should be between `fr` and `gl` alphabetically speaking, though I'm not sure it matters.